### DISCUSSION TITLE
Conductor 3 documentation + migration path?

### DISCUSSION THREAD

**Discussion by james-deee** [Netflix/conductor#2294] [Q&A]
Hello, I was wondering if there was some more specific documentations on what is changing between Conductor v3 versus v2. Also, are there documents on what is needed to migrate? Can you even migrate, or do you need to just start fresh with v3?

Thanks.

**Comment by apanicker-nflx** [comment]
The main documentation largely still applies to the project as the core modules and functionality are still the same. You can find more details in the [changelog](https://github.com/Netflix/conductor/blob/main/CHANGELOG.md). We will be updating the documentation with to better reflect the changes soon.
